test/language/expressions/exponentiation/exponentiation-operator.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2015 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Rick Waldron
+email: waldron.rick@gmail.com
+es7id: pending
+description: >
+    Performs exponential calculation on operands. Same algorithm as %MathPow%(x, y)
+---*/
+
+
+assert.sameValue(2 ** 3, 8, "Expected (3 * 2 ** 3) to be 8");
+assert.sameValue(3 * 2 ** 3, 24, "Expected (3 * 2 ** 3) to be 24");
+
+var x = 2;
+assert.sameValue(2 ** ++x, 8, "Expected (2 ** ++x) to be 8");
+assert.sameValue(2 ** -1 * 2, 512, "Expected (2 ** -1 * 2) to be 1");
+assert.sameValue(2 ** (3 ** 2), 512, "Expected (2 ** (3 ** 2)) to be 512");
+assert.sameValue(2 ** 3 ** 2, 512, "Expected (2 ** 3 ** 2) to be 512");
```

Comment:
Since we're testing operator precedence here, shouldn't these be in `test/language/expressions/exponentiation/operator-precedence.js` instead?